contracts/lib/LibBalances.sol

```diff
@@ -33,7 +33,7 @@ library Balances {
         uint256 lastUpdatedGasPrice;
     }
 
-    function netValue(Position calldata position, uint256 price)
+    function netValue(Position memory position, uint256 price)
         public
```

Comment:
not related to this PR, but we should make these functions `internal`

Reply:
Yeah there's a ticket for it - was going to be my next one!

Reply:
There's a ticket for this, I'll make it my next PR.